Add sole Directory Listing document (#1351)
zeit/docs: pages/docs/directory-listing.mdx

import Standard from '~/components/layout/standard'
import Note from '~/components/text/note'
import { GenericLink } from '~/components/text/link'

export const meta = {
  title: 'Directory Listing',
  description: 'what is the Directory Listing page and what does it do?',
  editUrl: 'pages/docs/directory-listing.mdx',
  lastEdited: '2019-10-01T20:09:33.000Z'
}

The Directory Listing page, which shows the contents within a direcory, is served when a particular path is both a directory and does not have an index file within it.

#### Disabling The Directory Listing

> To not show a Directory Listing page, add an index file, like `index.html`, which will then show that file instead when accessing the directory's path.

Reviewer: ```suggestion
To prevent showing a Directory Listing page, add an index file, such as `index.html`, which will then show that file instead when accessing the directory's path.
```

If you were expecting a file to be here, check your Build output from the Deployment Overview from your [ZEIT Dashboard](/dashboard).

> <Note>

Reviewer: I'd prefer this to be a section: "Adding a Custom 404 Page"

Even if we provide a small amount of text and then link to the routes documentation, I feel this would be much better than a note visually.

Author: I'd be happy to do this, however, there are better plans for this. I'd prefer to wait for the better solution. Would you rather I go with that structure now?

Reviewer: Yes please, I think it's a much better way to end the page than on a note.
